cli/command/system/info.go

```diff
@@ -128,6 +129,14 @@ func prettyPrintInfo(dockerCli *command.DockerCli, info types.Info) error {
 			}
 		}
 		fmt.Fprintf(dockerCli.Out(), " Node Address: %s\n", info.Swarm.NodeAddr)
+		managers := []string{}
+		for _, entry := range info.Swarm.RemoteManagers {
+			managers = append(managers, entry.Addr)
+		}
+		sort.Strings(managers)
+		if len(managers) > 0 {
+			fmt.Fprintf(dockerCli.Out(), " Manager Addresses: %s\n", strings.Join(managers, " "))
```

Comment:
I was wondering if there is lots of managers, this line could be some kind of unreadable. Is it OK for separate line for each one? Just my thoughts.